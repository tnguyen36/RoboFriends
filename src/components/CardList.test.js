import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import React from 'react';
import CardList from './CardList';

it('expect to render CardList component', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'John Snow',
            username: 'JohnJohn',
            email: 'john@gmail.com'
        }
    ]
    const tree = renderer.create(<CardList robots={mockRobots} />).toJSON();
    expect(tree).toMatchSnapshot();
})
