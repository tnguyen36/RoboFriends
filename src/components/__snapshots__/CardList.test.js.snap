// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
<div>
  <div
    className="bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
  >
    <img
      alt="robot"
      src="https://robohash.org/1?200x200"
    />
    <div>
      <h2>
        John Snow
      </h2>
      <p>
        john@gmail.com
      </p>
    </div>
  </div>
</div>
`;
